recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: cobra
+  version: "0.4.1"
+
+source:
+  fn: cobra-0.4.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/cobra/cobra-0.4.1.tar.gz
+  md5: a96799117abed427333bb57e007a8b82
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Since this package uses setuptools can you add `--single-version-externally-managed --record record.txt` to the install script (and `bld.bat`).